package.json

```diff
@@ -62,9 +62,7 @@
     "@storybook/react": "^5.2.6",
     "autoprefixer": "^7.2.6",
     "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.0.0",
     "chai": "^4.2.0",
-    "css-loader": "^0.28.11",
```

Comment:
In general I think i'm okay with the main part of this change, assuming that the storybook script/pages still works appropriately.